evm_loader/cli/src/main.rs

```diff
@@ -1167,12 +1171,11 @@ fn command_cancel_trx(
     Ok(())
 }
 
-fn command_neon_elf(
+fn read_elf_parameters(
     _config: &Config,
-    program_location: &str,
+    program_data: &[u8],
 ) {
-    let program_data = read_program_data(program_location).unwrap();
-    let program_data = &program_data[..];
+    //let program_data = &program_data[..];
```

Comment:
Please, remove this comment.

Reply:
Fixed

Reply:
Please, check it again. I see this comment in code.